generators/environment/templates/bin/start.sh

```diff
@@ -140,12 +140,10 @@ cmd "docker exec <%= machineName %>_${DOCKER_ENV}_www \"/var/www/bin/fix-perms.s
 
 # Install the site.
 if [ "$UPDATE" == 0 ]; then
-  # Errors in final steps of installation require --force to ensure bin/post-install.sh is run.
-  # Dev triggers a development build of Open Atrium
-  cmd "docker-compose run --rm cli \"grunt install --no-db-load --force\""
+  cmd "docker-compose run --rm grunt \"install --no-db-load\""
```

Comment:
The --force is set up because there are often inconsequential errors in early stage of new projects. I'm open to removing it, but we should probably do something else to guide developers to adding --force here if it's temporarily needed.